pkgs/tools/system/daemonize/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name    = "daemonize-${version}";
+  version = "1.7.5";
+
+  src = fetchurl {
+    url    = "https://github.com/bmc/daemonize/archive/release-${version}.tar.gz";
+    sha256 = "616220b8dc5721d93bd45e63b2617dbe07cd10a572da505405d64f640b98a06b";
+  };
+
+  meta = with stdenv.lib; {
```

Comment:
I can only test it on linux but according to the author supported platforms are

>  *  FreeBSD 4.x, 8.0-RELEASE, 8.1-RELEASE and 8.2-RELEASE
>  *  Red Hat Enterprise Linux 4 / CentOS 4
>  *  Solaris (SunOS 5.8, 5.10)
>  *  Fedora Core 5
>  *  Ubuntu 8, 9, 10, 11 and 12
>  *  Mac OS X 10.4 (Tiger) and 10.6 through 10.8.

Not sure what I should do...